<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>preferencesDialog</class>
 <widget class="QDialog" name="preferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="windowIcon">
   <iconset resource="eepskye.qrc">
    <normaloff>:/images/preferences.png</normaloff>:/images/preferences.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Language (requires restart)</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="locale_QB"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Check for updates on startup</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QCheckBox" name="startupCheck_er9x">
       <property name="text">
        <string>ersky9x</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="startupCheck_eepe">
       <property name="text">
        <string>eePskye</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Show Splash Screen</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="showSplash">
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Version to download</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QComboBox" name="downloadVerCB">
     <item>
      <property name="text">
       <string>ersky9x (Sky board)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ersky9xr (9XR-PRO)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>X9D (Taranis)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>X9DP (Taranis Plus)</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="9" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="10" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>Default Channel Order</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="channelorderCB">
     <property name="whatsThis">
      <string>Channel order

This is used by the templated to determine which channel goes to what number output.</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string notr="true">R E T A</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">R E A T</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">R T E A</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">R T A E</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">R A E T</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">R A T E</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">E R T A</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">E R A T</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">E T R A</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">E T A R</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">E A R T</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">E A T R</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">T R E A</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">T R A E</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">T E R A</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">T E A R</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">T A R E</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">T A E R</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">A R E T</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">A R T E</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">A E R T</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">A E T R</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">A T R E</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">A T E R</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>Default Stick Mode</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="stickmodeCB">
     <property name="whatsThis">
      <string>Mode selection:

Mode 1:
  Left stick:  Elevator, Rudder
  Right stick:  Throttle, Aileron

Mode 2:
  Left stick:  Throttle, Rudder
  Right stick:  Elevator, Aileron

Mode 3:
  Left stick:  Elevator, Aileron
  Right stick:  Throttle, Rudder

Mode 4:
  Left stick:  Throttle, Aileron
  Right stick:  Elevator, Rudder

</string>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <item>
      <property name="text">
       <string>Mode 1 (RUD ELE THR AIL)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mode 2 (RUD THR ELE AIL)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mode 3 (AIL ELE THR RUD)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mode 4 (AIL THR ELE RUD)</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="1">
    <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,10,10" columnminimumwidth="0,0,0">
     <item row="0" column="2">
      <widget class="QPushButton" name="er9x_dnld">
       <property name="text">
        <string>Download</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="er9x_ver_label">
       <property name="text">
        <string>r111</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="er9x_dnld_2">
       <property name="text">
        <string>Check</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_CurrentVersion">
     <property name="text">
      <string>Current Version - ersky9x</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QComboBox" name="defaultVersionCB">
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Default Model Version</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="eepskye.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>preferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>preferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>390</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
